q2_longitudinal/assets/index.html

```diff
@@ -2,6 +2,25 @@
 
 {% block content %}
 
+{% if errors %}
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="accordion-heading">
+      <h4 class="panel-title">
+        <a role="button" data-toggle="collapse" data-parent="#accordion" href="#table" aria-expanded="true" aria-controls="table">
+          WARNINGS (click here to collapse):
+        </a>
+      </h4>
+    </div>
+    <div id="table" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="accordion-heading">
+      <div class="alert alert-warning col-md-12">
+        <strong>{{ errors }}</strong>
```

Comment:
It would be a little more idiomatic to do something like:

```
{% for error in errors %}
<p><strong>{{ error }}</strong></p>
{% endfor %}
```

This would also mean you don't need the join on an HTML tag in your python code which make things nice an d separate.